vocabularies/HTML5.xml

```diff
@@ -49,6 +49,16 @@
         </Property>
       </ComplexType>
 
+      <Term Name="LinkTarget" Type="Edm.String" Nullable="true">
```

Comment:
How about making this a type definition with AllowedValues, describing (a subset of) the generic link targets `_self`, `_blank`, `_parent`, and `_top`?

Reply:
This is the HTML5 vocabulary, why not use the HTML5 terms here?